percy/percy.test.js

```diff
@@ -6,7 +6,7 @@ import puppeteer from 'puppeteer'
 const percySnapshot = require('@percy/puppeteer')
 
 const OTP_RR_TEST_CONFIG_PATH = '../percy/har-mock-config.yml'
-const { OTP_RR_PERCY_CALL_TAKER, OTP_RR_PERCY_MOBILE } = process.env
```

Comment:
The mobile env variable was used to enter mobile and leave mode -- it's more than changing the screen size, the user agent needs to be faked

Reply:
That was always the case.